src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidationResult.java

```diff
@@ -16,49 +16,40 @@
  * under the License.
  */
 
-package org.wso2.carbon.module.swiftiso20022.model;
+package org.wso2.carbon.module.swiftiso20022.validation.common;
 
 /**
  * Class to hold the error details.
  */
-public class ErrorModel {
-    private boolean isError;
+public class ValidationResult {
+    private boolean isValid;
     private String errorCode;
     private String errorMessage;
 
-    public ErrorModel(boolean isError, String errorCode, String errorMessage) {
-        this.isError = isError;
-        this.errorCode = errorCode;
-        this.errorMessage = errorMessage;
-    }
-
-    public ErrorModel(String errorCode, String errorMessage) {
-        this(true, errorCode, errorMessage);
-    }
-
-    public ErrorModel() {
+    public ValidationResult() {
+        this.isValid = true;
+        this.errorCode = null;
+        this.errorMessage = null;
```

Comment:
```suggestion
        this(true, null, null);
```

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/7f2058c938da71ec80755d296fd2d3d67541e678